### DISCUSSION TITLE
Range Selection

### DISCUSSION THREAD

**Discussion by Pratosh22** [ghiscoding/slickgrid-react#294] [Q&A] [answered]
How can i select multiple cells (range selection) ? Thanks :)

**Comment by ghiscoding** [comment, marked as answer]
Please ask these questions on Stack Overflow

```ts
function buttonClick() {
  const cellSelectionModel = this.reactGrid.slickGrid!.getSelectionModel();
  const range = new SlickRange(3, 3, 5, 4);
  cellSelectionModel?.setSelectedRanges([range]);

  // or
  cellSelectionModel?.setSelectedRanges([{ fromRow: 3, fromCell: 3, toRow: 5, toCell: 4 } as SlickRange]);
}
```

You can see the SlickRange in a live demo of Slickgrid-Universal [Example 19](https://ghiscoding.github.io/slickgrid-universal/#/example19)

![image](https://github.com/ghiscoding/slickgrid-react/assets/643976/da1599f5-938a-4c72-a889-a11199d2a490)
